packages/@angular/cli/blueprints/ng/files/protractor.conf.js

```diff
@@ -1,7 +1,6 @@
 // Protractor configuration file, see link for more information
 // https://github.com/angular/protractor/blob/master/lib/config.ts
 
-/*global jasmine */
```

Comment:
Isn't this applicable here in this file https://github.com/filipesilva/angular-cli/blob/8649b892c504b4b8acd79b467f03e9103d59595b/packages/%40angular/cli/blueprints/ng/files/protractor.conf.js#L28